### PR TITLE
Issues sample linked elements

### PR DESCRIPTION
**Pull request by dustinlebsock**
Adds Linked elements in the Summary tab of the issue sample:
image

### CODE REVIEW

File: src/frontend-samples/issues-sample/IssuesWidget.tsx [hunk 1]
```diff
@@ -180,6 +182,21 @@ const IssuesWidget: React.FunctionComponent = () => {
...
+    /** Don't refetch if we have already received the linked elements */
+    if (!iModelConnection || currentLinkedElements || !currentIssue)
+      return;
+
+    if (!currentIssue.item?.id) {
+      setLinkedElements([]);
+      return;
+    }
+
+    const elementKeySey = await IssuesApi.getElementKeySet(currentIssue.item.id);
```
Comment (williamkbentley) [inline src/frontend-samples/issues-sample/IssuesWidget.tsx:195]:
```suggestion
    const elementKeySet = await IssuesApi.getElementKeySet(currentIssue.item.id);
```

File: src/frontend-samples/issues-sample/IssuesApi.ts [hunk 2]
```diff
@@ -3,14 +3,39 @@
...
+      return new KeySet();
+
+    const keySetJSON: KeySetJSON = JSON.parse(elementsId);
+    return KeySet.fromJSON(keySetJSON);
+  }
+
+  public static async getElementInfo(iModel: IModelConnection, keySet: KeySet): Promise<LabelWithId[]> {
+    const instanceKeys: InstanceKey[] = [];
+    keySet.instanceKeys.forEach((currentIds: Set<string>, key: string) => {
+      currentIds.forEach ((value: string) => { instanceKeys.push ({className: key, id: value}); });
```
Comment (williamkbentley) [inline src/frontend-samples/issues-sample/IssuesApi.ts:31]:
```suggestion
      currentIds.forEach((value: string) => { instanceKeys.push ({className: key, id: value}); });
```

**Review by williamkbentley — CHANGES_REQUESTED** [review]
- Add "- Get list of elements linked to an issue", or similar to README.

File: src/frontend-samples/issues-sample/IssuesApi.ts (hunk 2 above)
Comment (dustinlebsock) [reply src/frontend-samples/issues-sample/IssuesApi.ts]: space removed.

File: src/frontend-samples/issues-sample/IssuesWidget.tsx (hunk 1 above)
Comment (dustinlebsock) [reply src/frontend-samples/issues-sample/IssuesWidget.tsx]: renamed.

**Review by williamkbentley — APPROVED** [review]

**Pull request merged**